src/domain/orders/details/swap/create.tsx

```diff
@@ -0,0 +1,343 @@
+import React, { useState, useEffect, useContext } from "react"
+import Modal from "../../../../components/molecules/modal"
+import Button from "../../../../components/fundamentals/button"
+import Select from "../../../../components/molecules/select"
+import Medusa from "../../../../services/api"
+import { filterItems } from "../utils/create-filtering"
+import { getErrorMessage } from "../../../../utils/error-messages"
+import RMASelectProductTable from "../../../../components/organisms/rma-select-product-table"
+import Spinner from "../../../../components/atoms/spinner"
+import RMAShippingPrice from "../../../../components/molecules/rma-select-shipping"
+import RMAReturnProductsTable from "../../../../components/organisms/rma-return-product-table"
+import InfoTooltip from "../../../../components/molecules/info-tooltip"
+import CheckIcon from "../../../../components/fundamentals/icons/check-icon"
+import LayeredModal, {
+  LayeredModalContext,
+} from "../../../../components/molecules/modal/layered-modal"
+import RMASelectProductSubModal from "../rma-sub-modals/products"
+
+const removeNullish = (obj) =>
+  Object.entries(obj).reduce((a, [k, v]) => (v ? ((a[k] = v), a) : a), {})
+
+const extractPrice = (prices, order) => {
+  let price = prices.find((ma) => ma.region_id === order.region_id)
+
+  if (!price) {
+    price = prices.find((ma) => ma.currency_code === order.currency_code)
+  }
+
+  if (price) {
```

Comment:
again use price utils so we ensure that it works with won, yen, and all the currencies that should be divided by 1000 😄